### PR TITLE
Refresh topic about installing to GAC

### PR DESCRIPTION
**Pull request by gewarren**
Removes references to VS 2012 and 2013. Also removes InstallShield section since it's not included with VS 2017. Adds info about the WiX Toolset extension for VS 2017.

Part of the fix for #7172.

### CODE REVIEW

**Comment by gewarren** [comment]
@rpetrusha this is good to go for review. @Mikejo5000 thanks for the info about this!

File: docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md [hunk 1]
```diff
@@ -1,79 +1,55 @@
...
+
+In this command, *assembly name* is the name of the assembly to install in the global assembly cache.
+
+The following example installs an assembly with the file name `hello.dll` into the global assembly cache.
+
+```shell
+gacutil -i hello.dll
+```
+
 > [!IMPORTANT]
```
Comment (rpetrusha) [inline docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md]: Side-by-side notes (Important followed by Note) is not optimal. This also is important -- without a strong-named assembly, there's no point in reading further. I'd move this immediately after the article's first paragraph.

File: docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md [hunk 2]
```diff
@@ -1,79 +1,55 @@
...
-# How to: Install an Assembly into the Global Assembly Cache
-There are two ways to install a strong-named assembly into the global assembly cache (GAC):  
-  
+# How to: Install an assembly into the global assembly cache
+
+There are two ways to install a strong-named assembly into the global assembly cache (GAC).
+
+## Windows Installer
+
+[Windows Installer](/windows/desktop/Msi/installation-of-assemblies-to-the-global-assembly-cache), the Windows installation engine, is the recommended and most common way to add assemblies to the global assembly cache. Windows Installer provides reference counting of assemblies in the global assembly cache, plus other benefits. You can use the [WiX Toolset extension for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension) to create an installer package for Windows Installer.
```
Comment (rpetrusha) [inline docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md]: this is completely unrelated to your PR, but "recommended and most common" is a jarring phrase (at least I think so). I'd change it to just "recommended".
another unrelated nit: no comma after cache.

**Review by rpetrusha — APPROVED** [review]
I left one comment related to your PR and two nits unrelated to it, @gewarren. You can merge when you're ready.

**Pull request merged**